angular/src/app/containers/form/form.html

```diff
@@ -0,0 +1,39 @@
+<form [formGroup]="selection">
```

Comment:
Nu in Angular 2+ Reactive Forms de standaard zijn lijkt het me goed om de mogelijke waardes van een `<select>` eveneens buiten het template te houden.